src/store/actions/calendarActions.ts

```diff
@@ -46,13 +55,16 @@ function initialCalendarData() {
         dateArray: [],
         today: null,
         isRefreshing: false,
+        selectedDate: null,
     };
     const today: Date = new Date();
+    const selectedDate = null;
     const defaultDateArray = createDateArray(
-        new Date(today.getFullYear(), today.getMonth() - 12, 1),
-        new Date(today.getFullYear(), today.getMonth() + 12, 1),
+        new Date(today.getFullYear(), today.getMonth() - 6, 1),
```

Comment:
I see that you have to change these numbers a lot to work with the carousal. maybe you can store this number as a separate constant somewhere else. I feel that if you keep changing each manually, you could forget and cause bugs